README.md

```diff
@@ -136,7 +136,7 @@ module.exports = {
         test: /\.less$/,
         use: ExtractTextPlugin.extract({
           fallbackLoader: 'style-loader',
-          loaders: [
```

Comment:
Take time to read my code, the Webpack 2 docs and run the changes yourself before making comments like. I'm using Webpack ^2.2.0, and the "valid" syntax you're claiming is not valid on the docs.